enterprise/server/util/oci/oci.go

```diff
@@ -37,6 +40,12 @@ import (
 	bspb "google.golang.org/genproto/googleapis/bytestream"
 )
 
+const (
+	// resolveImageDigestLRUMaxEntries limits the number of entries in the image-tag-to-digest cache.
+	resolveImageDigestLRUMaxEntries = 1000
+	resolveImageDigestLRUDuration   = time.Minute * 15
```

Comment:
uber nit: maybe just personal preference, but `15 * time.Minute` reads more naturally to me, like "15 minutes"

```suggestion
	resolveImageDigestLRUDuration   = 15 * time.Minute
```